tools/src/main/scala/org/apache/spark/sql/rapids/tool/qualification/QualAppInfo.scala

```diff
@@ -228,8 +255,22 @@ class QualAppInfo(
         val existingIssues = sqlIDtoProblematic.getOrElse(sqlID, Set.empty[String])
         sqlIDtoProblematic(sqlID) = existingIssues ++ issues
       }
+      // Get the write data format
+      if (node.name.contains("InsertIntoHadoopFsRelationCommand")) {
+        val writeFormat = node.desc.split(",")(2)
+        writeDataFormat += writeFormat
+      }
     }
   }
+
+  def writeFormatNotSupported(writeFormat: ArrayBuffer[String]): String = {
+    val supportedWriteDataFormat = Array("parquet", "orc")
+    //Filter unsupported write data format
+    val unSupportedWriteFormat = writeFormat.map(x => x.toUpperCase.trim).filterNot(
```

Comment:
just do a to lower here, the supportedWritedataFormat is controlled by us and already lower,